.github/workflows/gemfiles-getter.yml

```diff
@@ -0,0 +1,35 @@
+name: Retrieve Gemfiles
+
+on:
+  workflow_call:
+
+jobs:
+  fetch:
+    name: Retrieve Gemfiles
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Remove existing Gemfile, Gemfile.lock, gemspec and /lib
+      run: rm -rf Gemfile Gemfile.lock relaton-cli.gemspec lib/
+    - name: mkdir bibxml-service
+      run: mkdir bibxml-service
+    - name: Checkout bibxml-service
+      uses: actions/checkout@v3
```

Comment:
instead of cloning gem manually you can specify exact version from git with bundler here is example:
`bundle remove relaton-cli`
`bundle add relaton-cli --github 'relaton/relaton-cli' --ref 'gemfilelock'`

More information here:
 - https://bundler.io/v2.4/man/bundle-add.1.html
 - https://bundler.io/guides/git.html

Reply:
This makes sense, thank you. I'll have a look